barretenberg/cpp/CMakePresets.json

```diff
@@ -268,7 +268,7 @@
       "displayName": "Build with fuzzing",
       "description": "Build default preset but with fuzzing enabled",
       "inherits": "clang16-dbg",
-      "binaryDir": "build-fuzzing",
+      "binaryDir": "build-fuzzing-asan",
```

Comment:
Could you create a separate asan build?

Reply:
There is no need to change the directory without adding ASAN by default to the build. So either this change is not needed or it would be beneficial to create a separate build with ASAN, so it is easy to reconfigure